### PR TITLE
LEAF 4464 - general 508 fixes

### PR DESCRIPTION
**Pull request by aerinkayne**
Addresses several issues noted in the most recent 508 audit and other issues uncovered during testing

- Wide pass to resolve issues associated with icon alt text, redundant title attributes, labels, and page titles. 
- Minor updates to portal admin header to improve consistency.
- Improves info for and feedback relating to validation of date format questions.
- 'Aria-expanded' attributes and elements with status roles added to several controller elements in the LEAF Inbox and new Form Editor to provide feedback about submenu states and certain actions (screen reader needed to test).
- Fixes modal focus drop in the new form editor and adds focus tracking when using the click to move buttons in new form editor 
- Fixes a keyboard trap in the new form editor Programmer dialog.
- UX / HD related
New form editor, indicator editing modal: 'default answer' input area does not show for attachment formats or serialized inputs (grid, checkboxes, multiselect)

Impact / Testing

Most changes should not be highly noticeable to users.  Regression and API tests to help rule out unintended changes.  

- Confirm that portal admin nav functions as expected.
- LEAF Inbox: Screen reader should announce submenu states and the outcome of toggle and view selections.  NVDA is a free screen reader that can be used for local testing (https://www.nvaccess.org/download/).
- New form editor: When a modal is closed, focus is returned to the button user clicked to open the modal.
- New form editor, Programmer: user can successfully tab through code editor sections of the modal by following the provided instructions
- New form editor: If using click to move buttons to move questions (click the drag tab on the left side of each question to show them), focus is returned back to the clicked button after the item is moved.
- New form editor, indicator editing: The default answer input field does not show for formats that are none, image or file uploads, raw data, grid, checkboxes or multiselect
- Need to know turns on if a form with sensitive questions is imported
- Form Editor Form History modal shows more details about condition changes

### CODE REVIEW

File: LEAF_Nexus/templates/print_subindicators_ajax.tpl [hunk 1]
```diff
@@ -91,10 +91,10 @@
             <!--{foreach from=$indicator.options item=option}-->
                     <input type="hidden" name="<!--{$indicator.indicatorID|strip_tags|escape}-->[<!--{$idx}-->]" value="no" /> <!-- dumb workaround -->
                     <!--{if $option == $indicator.data[$idx]}-->
-                        <br /><img class="print" src="images/checkbox-yes.png" alt="checked" />
+                        <br /><img class="print" src="images/checkbox-yes.png" alt="" />
                         <!--{$option|strip_tags|escape}-->
                     <!--{else}-->
-                        <br /><img class="print" src="images/checkbox-no.png" alt="not checked" />
```
Comment (shaneodd) [inline LEAF_Nexus/templates/print_subindicators_ajax.tpl:97]: missing double quote?

File: docker/vue-app/src/form_editor/components/dialog_content/AdvancedOptionsDialog.js [hunk 2]
```diff
@@ -163,35 +191,38 @@ export default {
...
                         <td>Escape Full Screen mode</td>
                     </tr>
-                </table><br />
+                </table>
+                <div style="font-size:14px;">
+                    Within the code editor, tab enters a tab character. If using the keyboard to navigate, press escape followed by tab to exit the editor.
+                </div><br />
                 <div class="save_code">
-                    html (for pages where the user can edit data): 
-                    <button type="button" id="btn_codeSave_html" class="btn-general" @click="saveCodeHTML" title="Save Code">
```
Comment (shaneodd) [inline docker/vue-app/src/form_editor/components/dialog_content/AdvancedOptionsDialog.js:186]: same here is this a graphical glitch?

File: docker/vue-app/src/form_editor/components/dialog_content/AdvancedOptionsDialog.js [hunk 3]
```diff
@@ -163,35 +191,38 @@ export default {
...
+                    <label for="codemirror_html_label">html (for pages where the user can edit data):</label>
+                    <button type="button" id="btn_codeSave_html" class="btn-general" @click="saveCodeHTML" aria-label="save html code">
                         <img id="saveIndicator" :src="libsPath + 'dynicons/svg/media-floppy.svg'" alt="" />
                         &nbsp;Save Code<span id="codeSaveStatus_html"></span>
                     </button>
                 </div>
                 <textarea id="html">{{html}}</textarea><br />
                 <div class="save_code">
-                    htmlPrint (for pages where the user can only read data): 
-                    <button  type="button" id="btn_codeSave_htmlPrint" class="btn-general" @click="saveCodeHTMLPrint" title="Save Code">
```
Comment (shaneodd) [inline docker/vue-app/src/form_editor/components/dialog_content/AdvancedOptionsDialog.js:194]: same here is this a graphical glitch?

**Review by shaneodd — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by KCN8 — APPROVED** [review]

**Review by jampaul3 — APPROVED** [review]
